src/Log/Helpers/Environment.php

```diff
@@ -28,4 +28,14 @@ public static function isLogsPage()
 
         return false;
     }
+
+    /**
+     * @unreleased
+     *
+     * @return bool
+     */
+    public static function isLogEnabled()
```

Comment:
Last thing: let's make this clearer: `isDebugLoggingEnabled()`

Reply:
👍  91fb46d8